api/envoy/config/filter/http/ext_authz/v2/ext_authz.proto

```diff
@@ -24,7 +24,7 @@ option (udpa.annotations.file_status).package_version_status = FROZEN;
 // External Authorization :ref:`configuration overview <config_http_filters_ext_authz>`.
 // [#extension: envoy.filters.http.ext_authz]
 
-// [#next-free-field: 11]
+// [#next-free-field: 12]
```

Comment:
@envoyproxy/api-shepherds we have a product need for this to be available sooner rather than later, would you be willing to accommodate a v2 freeze override?

Reply:
Can I get sliding window deprecations  in exchange? ;)

Fine with me.

Reply:
Please let me know anything else I need to complete to get approved :)

Reply:
I think we're good to go on the v2 freeze override, resolved.